Source/WebCore/Modules/mediasession/MediaSessionCoordinator.cpp

```diff
@@ -47,7 +48,7 @@ namespace WebCore {
 
 static const void* nextCoordinatorLogIdentifier()
 {
-    static uint64_t logIdentifier = cryptographicallyRandomNumber();
+    static uint64_t logIdentifier = cryptographicallyRandomNumber<uint32_t>();
```

Comment:
Left this as is, generating a 32-bit random number and storing it in a 64-bit global. New function name makes that a lot more obvious. Not sure if it’s a mistake.